src/main/java/reposense/util/FileUtil.java

```diff
@@ -84,8 +85,11 @@ public static void zipFoldersAndFiles(List<Path> pathsToZip,
 
     /**
      * Writes the JSON file representing the {@code object} at the given {@code path}.
+     *
```

Comment:
can remove all these blank lines to adhere to current standard.

Reply:
Just to clarify - the OSS Java standard says we should keep an empty line between the description and parameter section of the Javadoc.

![image](https://user-images.githubusercontent.com/35621759/69494923-095f2800-0efc-11ea-9622-84c451bfe748.png)

Should I still remove the empty lines?/

Reply:
@eugenepeh, let me know if I should still remove the empty lines.
I followed the Java OSS standard.

Reply:
Good catch, however do observe that this does not make any difference on IntelliJ IDE itself (the separation is done on the IDE level), hence should be done on a case by case basis. 

For single line comments that having a new line or not will not affect its readability, it is preferred to have it removed so that more code can be displayed on the screen.

Reply:
Ok, I will remove the newlines for cases where the description block only has a single line.

Reply:
@eugenepeh, I have removed the empty lines for Javadocs that have only a single line in the description block. Thanks for taking the time to review this PR.